### DISCUSSION TITLE
How

### DISCUSSION THREAD

**Discussion by dhust** [parcel-bundler/parcel#8562] [Q&A]
In JavaScript, I'm trying to redirect to a webpage in localhost. Doing `http://localhost:1234/home'` doesn't work. All it does is redirect back to the main `index.html` page. In fact, any `/*` redirects back to the login. In my `dist` folder I have `home.html`, `index.html`, and `script.dev.js`. I just want to redirect to `home.html` Any help would be greatly appreciated.

`const promise = account.createOAuth2Session('google', 'http://localhost:1234/home', 'http://google.com');`

**Comment by mischnic** [comment]
I think you have to include the extension, so `http://localhost:1234/home.html`
